lib/org/chromium/webidl/ast/Definition.js

```diff
@@ -21,8 +21,8 @@ foam.CLASS({
       factory: function() { return this.definition.getName(); },
     },
     {
-      class: 'Array',
-      of: 'String',
+      class: 'FObjectArray',
+      of: 'org.chromium.webidl.BaseIDLFile',
```

Comment:
Aside: Perhaps we should file an issue for `BaseIDLFile` to contain an interface for extracting implementation-specific ID or other metadata. I'm also a little concerned about pulling them back out of Datastore when they are of different "Kinds" (in Datastore parlance), but this is probably fine for now.